src/mantine-demos/src/demos/core/Grid/Grid.demo.order.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Grid } from '@mantine/core';
+import { ColWrapper as Col } from './_col-wrapper';
+
+const code = `
+import { Grid } from '@mantine/core';
+
+function Demo() {
+  return (
+    <Grid>
+      <Col span={3} order={2}>2</Col>
```

Comment:
Please add responsive props (let's say `orderSm` and `orderLg` to the demo)

Reply:
Sounds good - should I also use the [`useMediaQuery`](https://mantine.dev/hooks/use-media-query/) hook here to relabel the children based on their new order?